packages/miniflare/scripts/build.mjs

```diff
@@ -186,8 +187,10 @@ async function buildPackage() {
 			"esbuild",
 		],
 		plugins: [embedWorkersPlugin],
-		logLevel: watch ? "info" : "warning",
-		watch,
+		logLevel: "warning",
```

Comment:
disabled watch mode here because we updated esbuild and it didn't have the same api. I'll being it back later.